tutorials/VI_in_BM.ipynb

```diff
@@ -0,0 +1,2009 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    from beanmachine.ppl.experimental.vi.variational_infer import (
</pre><p>We are <a href="https://github.com/facebookresearch/beanmachine/blob/main/src/beanmachine/ppl/inference/vi/variational_infer.py" target="_blank">moving VI out of experimental</a>, so don't forget to update the import path before merging this tutorial in :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>